server/app/assets/javascripts/preview.ts

```diff
@@ -210,17 +213,21 @@ class PreviewController {
               `.${PreviewController.QUESTION_MULTI_OPTION_VALUE_CLASS}`,
             ),
           )
-      optionText.innerText = configuredOption
+
+      // Objects in javascript are passed by value so editing optionText.innerHTML
+      // modifies newPreviewOption
+      optionText.innerHTML = DOMPurify.sanitize(formatText(configuredOption), {
```

Comment:
Even though we're running the text through the sanitizer in `formatText`, we need to run it through here as well to satisfy the eslint check.

Reply:
(question, blocking) When you say option does that mean an actual html `option` element or something else? Just want to double check since it's not allow to have elements inside the `option` element.

Reply:
it's a div 
![image](https://github.com/civiform/civiform/assets/24319951/ef3f117a-241b-48a7-b17c-0fd0979ff480)